ui/editor.py

```diff
@@ -5,13 +5,37 @@ class NotebookEditor():
     NotebookEditor defines the UI for a simple notebook editor using tkinter. It
     optionally takes a DistributedNotebook object as input to synchronize with.
     '''
-    def __init__(self, notebook=None):
+    def __init__(self, client=None):
         self.root = tk.Tk()
-        self.root.title("Collaborative Notebook Editor")
-        self.notebook = notebook
+        self.root.title(client.name if client is not None else "Untitled Notebook")
+        self.client = client
         self.cells = []
+        if self.client is not None:
+            for peer in self.client.get_peers():
```

Comment:
I guess this would make it tricky to add a new peer later on that wasn't in the initial peer set. Any ideas about how we could handle that use case?

Reply:
I think we could just have `listen` watch for notebooks owned by peers it doesn't know about and update the peers list as well as the UI element.